### PR TITLE
Return response from StaticLargeObjectApi.delete

### PR DESCRIPTION
**Pull request by andrewgaul**

### CODE REVIEW

**Comment by andrewgaul** [comment]
@zack-shoylev Please review.  Related to #878.

File: apis/openstack-swift/src/test/java/org/jclouds/openstack/swift/v1/features/StaticLargeObjectApiLiveTest.java [hunk 1]
```diff
@@ -88,11 +94,27 @@ public void testReplaceManifest() throws Exception {
...
          assertEquals(api.getContainerApi(regionId).get(containerName).getObjectCount(), 0);
       }
    }
 
+   public void testDeleteSinglePartObjectWithMultiPartDelete() throws Exception {
+      String objectName = "testDeleteSinglePartObjectWithMultiPartDelete";
+      for (String regionId : regions) {
+         api.getObjectApi(regionId, containerName).put(objectName, newByteSourcePayload(ByteSource.wrap("swifty".getBytes())));
+         DeleteStaticLargeObjectResponse resp = api.getStaticLargeObjectApi(regionId, containerName).delete(objectName);
+         assertThat(resp.status()).isEqualTo("400 Bad Request");
```
Comment (andrewgaul) [inline apis/openstack-swift/src/test/java/org/jclouds/openstack/swift/v1/features/StaticLargeObjectApiLiveTest.java]: This illustrates the problem we encountered in #878.

Comment (zack-shoylev) [inline apis/openstack-swift/src/test/java/org/jclouds/openstack/swift/v1/features/StaticLargeObjectApiLiveTest.java]: Ok, looks good.

But shouldn't the mock test for StaticLargeObjectApi also change to account for the response change?

Comment (andrewgaul) [inline apis/openstack-swift/src/test/java/org/jclouds/openstack/swift/v1/features/StaticLargeObjectApiLiveTest.java]: Augmented mock tests in the updated commit.

**Comment by zack-shoylev** [comment]
+1

**Pull request merged**